incubator/command-dump/src/test/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommandTest.java

```diff
@@ -50,142 +55,239 @@ public class ZillaDumpCommandTest
     private ZillaDumpCommand command;
 
     @BeforeAll
+    @SuppressWarnings("checkstyle:methodlength")
     public static void generateStreamsBuffer()
     {
         StreamsLayout streamsLayout = new StreamsLayout.Builder()
             .path(Paths.get(baseDir, "engine").resolve("data0"))
             .streamsCapacity(8 * 1024)
             .readonly(false)
             .build();
-
         RingBuffer streams = streamsLayout.streamsBuffer();
-
         MutableDirectBuffer frameBuffer = new UnsafeBuffer(new byte[1024 * 8]);
 
-        BeginFW begin = new BeginFW.Builder().wrap(frameBuffer, 0, frameBuffer.capacity())
+        SignalFW signal1 = new SignalFW.Builder().wrap(frameBuffer, 0, frameBuffer.capacity())
             .originId(0)
             .routedId(0)
             .streamId(0)
             .sequence(0)
             .acknowledge(0)
             .maximum(0)
-            .affinity(0)
+            .timestamp(0x0000000000000001L)
+            .traceId(0x0000000000000000L)
+            .cancelId(0x0000000000007701L)
+            .signalId(0x00007702)
+            .contextId(0x00007703)
             .build();
+        streams.write(SignalFW.TYPE_ID, signal1.buffer(), 0, signal1.sizeof());
 
-        streams.write(BeginFW.TYPE_ID, begin.buffer(), 0, begin.sizeof());
-
-        BeginFW begin2 = new BeginFW.Builder().wrap(frameBuffer, 0, frameBuffer.capacity())
+        String hello = "Hello World!";
+        byte[] helloBytes = hello.getBytes(StandardCharsets.UTF_8);
```

Comment:
Use `DirectBuffer helloBuf = new String8FW("Hello, World!").value();` to get a `DirectBuffer` of exact capacity.

Use `.payload(helloBuf, 0, helloBuf.capacity())` in builder instead of creating `OctetsFW`.

Same feedback for other payloads below.

Reply:
👍